bigtable-client-core-parent/bigtable-hbase-integration-tests-common/src/test/java/com/google/cloud/bigtable/hbase/AbstractTestFilters.java

```diff
@@ -2180,6 +2223,24 @@ protected Table addDataForTesting(String rowKeyPrefix, byte[] qualA)
     return table;
   }
 
+  private void assertRowKeysWithRegex(Table table, String[] rowRegEx,
+      String rowPrefix, String[] rowKeys, int[] expected) throws IOException {
+    FilterList filtersList = new FilterList(FilterList.Operator.MUST_PASS_ONE);
+    for (String condition : rowRegEx) {
+      filtersList.addFilter(new RowFilter(CompareOp.EQUAL,
+          new RegexStringComparator(rowPrefix + condition)));
+    }
+    Scan scan = new Scan();
+    scan.setFilter(filtersList);
+    try (ResultScanner scanner = table.getScanner(scan)) {
+      Result[] results = scanner.next(10);
+      for (int i = 0; i < results.length; i++) {
+        Assert.assertArrayEquals(results[i].getRow(),
+            Bytes.toBytes(rowPrefix + rowKeys[expected[i]]));
+      }
+    }
```

Comment:
please add another test to make sure that the `scanner.next()` is empty.

Reply:
Thanks for the comment, Added a check for the scanner.next().